test/utils.jl

```diff
@@ -326,14 +326,14 @@ end
 
 @testset "Stacking" begin
   x = randn(3,3)
-  stacked = stack([x, x], dims=2)
+  stacked = Flux.MLUtils.stack([x, x], dims=2)
```

Comment:
```suggestion
  stacked = MLUtils.stack([x, x], dims=2)
```
We'd like to avoid reaching into modules to get dependencies which are directly accessible in the project, primarily because Flux may stop importing MLUtils (or any other dep) at the top level at some point. This, doing the same below and adding an import for MLUtils should do the trick.

Reply:
I've made the changes and all tests passed.